src/detectors/RPOTS/RomanPotsReconstruction_factory.cc

```diff
@@ -19,8 +19,14 @@ namespace eicrecon {
 
 	m_log = app->GetService<Log_service>()->logger("ForwardRomanPotRecParticles");
 
-	/*
-        auto id_spec = detector->readout(m_readout).idSpec();
+	m_readout = "ForwardRomanPotHits";
+	app->SetDefaultParameter("RPOTS:ForwardRomanPotHits:readoutClass", m_readout);
+	m_geoSvc = app->GetService<JDD4hep_service>();
+
+
+	if(m_readout.empty()){ std::cout << "READOUT IS EMPTY!" << std::endl;  return; }
```

Comment:
Could you please use m_log instead of std::cout

Reply:
Fixed.